impala/util.py

```diff
@@ -214,9 +214,28 @@ def get_all_matching_cookies(cookie_names, path, resp_headers):
             cookie_headers = resp_headers.get_all('Set-Cookie')
             for header in cookie_headers:
                 cookies.load(header)
+        return cookies
     except Exception:
         return None
```

Comment:
I think cookies.load can throw if it encounters an unexpected format.

Reply:
Usually ignoring errors is going to make troubleshooting more difficult, but, in this case, the error is being ignored already in the `master` branch code.  Thus, we cannot change it as that would break backwards compatibility.